environment/interpretation/lane_detector/launch/module.launch

```diff
@@ -3,17 +3,17 @@
 <launch>
     
     <node pkg="lane_detector" type="lane_detector" name="lane_detector1" args="lane_detector1" output="screen" cwd="node">
-        <param name="debug_mode" type="int" value="0"/>
-        <param name="ipt_offsets_file" type="str" value="../data/ipt_offsets1.txt"/>                
+        <param name="debug_mode" type="int" value="8"/>
+        <param name="ipt_offsets_file" type="str" value="ipt_offsets1.txt"/>                
         <param name="map_size" type="int" value="1000"/>        
         <param name="published_topic_name" type="str" value="/lane_detector1/lanes"/>        
         <param name="subscribed_topic_name" type="str" value="/logitech_camera1/image"/>
-        <param name="training_data_file" type="str" value="../data/Test4"/>
+        <param name="training_data_file" type="str" value="Samples1"/>
```

Comment:
use `"$(find <package_name>)/Samples1"` instead. Other wise this launch file won't work from any other dir. It will check for `Samples1` only at the dir where you run the launch file. Do this for all other paths.